boto3/docs/method.py

```diff
@@ -59,3 +59,14 @@ def document_model_driven_resource_method(
         new_return_section.write(
             ':returns: %s' % return_description)
         new_return_section.style.new_line()
+
+
+def _method_returns_collection(resource):
+    if resource.path and '[]' in resource.path:
```

Comment:
I am pretty sure we can remove this check completely. Based on how resource models work, if a path with a ``[]`` is specified for that resource than at least one of the identifiers must have the ``[]`` in its path as well.